dsse/sign.go

```diff
@@ -77,16 +78,27 @@ type SignVerifier interface {
 }
 
 // EnvelopeSigner creates signed Envelopes.
-type EnvelopeSigner struct {
+type envelopeSigner struct {
```

Comment:
Is there a reason why we unexport this type suddenly?

Reply:
I _think_ it's okay given the `New...` API, but I don't have a strong opinion here. cc @houdini91

Reply:
Hmm, I think I'm now leaning towards keeping this exported @houdini91.